docs/standard-library/rest/reference/js-api/interfaces/ResourceOperation.md

```diff
@@ -0,0 +1,22 @@
+[Documentation](../index.md) / ResourceOperation
```

Comment:
This is a lot of new files, do we want:
1. To actually include this in the committed repo? If not will have to make an extra step for generating the latest version vs next
2. This is using the markdown theme for typedoc, but we could get a more compact output like we have for the cadl if we wrote our own markdown theme

Reply:
I would say no to having this checked in (feels like build output to me).

Reply:
Should we then also not have the cadl ref docs be included

Reply:
That would be good. Let's use git ignore for all build output ref folders. For docs in the versioned release we do a manual step check-in or automate it with a small script easily.